### ISSUE TITLE
RC Test: Energy Skate Park: Basics 1.1.0-rc.1

### ISSUE THREAD

**Issue by samreid** [phetsims/tasks#550]
Energy Skate Park: Basics 1.1.0-rc.1 is ready for testing (as prepared in https://github.com/phetsims/energy-skate-park-basics/issues/336 )

**Link to sim:** http://www.colorado.edu/physics/phet/dev/html/energy-skate-park-basics/1.1.0-rc.1/energy-skate-park-basics_en.html

**Link to iFrame:** http://www.colorado.edu/physics/phet/dev/html/energy-skate-park-basics/1.1.0-rc.1/energy-skate-park-basics_en-iframe.html

**Test Matrix:** https://docs.google.com/spreadsheets/d/1D-qTre2wNUoDpCMvTtytWfW9pN72YITkHf2Qe0-P5uc/edit

Please do a full retest of this sim to make sure that no new issues were introduced.  This is also the first release candidate with support for internationalization, so please use ?stringTest appropriately.  Dev testing for an identical version was done in #540. If you notice any issues that are not listed in https://github.com/phetsims/energy-skate-park-basics/issues, please create a new issue and reference this one.

**Redeploy General Tests** (copied from https://github.com/phetsims/tasks/issues/528 )
The following tests should also be performed as part of this testing
- [x] stringTest=double
- [x] stringTest=long
- [x] stringTest=X (short strings)
- [x] stringTest=rtl (right-to-left)
- [x] stringTest=xss (should not redirect, OK to look bad, test on one desktop platform)
- [x] showPointerAreas (touchArea=red, mouseArea=blue)
- [x] Full screen test
- [x] Reference issues to RC test task

@ariel-phet please make sure I've provided all the necessary details, prioritize and reassign.

**Comment by ariel-phet** [comment]
@BryceAG @brroberts1231 @elisemorgan @bryo5363 @phet-steele please makes this the next testing priority after #544 is complete.

**Comment by ariel-phet** [comment]
Bumping to top priority

**Comment by phet-steele** [comment]
@ariel-phet @samreid matrix complete, with outstanding crashing issues on iPads

**Comment by samreid** [comment]
It looks like all issues are tracked elsewhere, closing.
